app/labor/cache_buster.rb

```diff
@@ -10,6 +10,8 @@ def bust(path)
                   headers: { "Fastly-Key" => ApplicationConfig["FASTLY_API_KEY"] })
     HTTParty.post("https://api.fastly.com/purge/https://dev.to#{path}?i=i",
                   headers: { "Fastly-Key" => ApplicationConfig["FASTLY_API_KEY"] })
+  rescue URI::InvalidURIError => e
+    Rails.logger.error("Trying to bust cache of an invalid uri: #{e}")
```

Comment:
My only concern with this is that this will silently fail, I'm more inclined into keeping this broken and work on the actual solution from #2934 (either by quoting/encoding or by using the fastly client)

Reply:
I'm also concerned with the silent failing. 
It doesn't make much sense to bust cache for such urls because they don't work on DEV.
Example: this [post](https://dev.to/sinaezekiel/team-members-needed-349k) and the [tag link](https://dev.to/t/#WavesRide%20#Solidity%20#Ethereum). The proper solution requires tags validation/parsing when a user inputs them in an unintended way. I suppose that we'll never work on a fix because it's for the "future", but resources are wasted and I also spend time looking through these dj errors. 
I'm also concerned with `rescue StandardError`s inside the `CacheBuster`.

Reply:
I think using `fastly-ruby` is a good idea.

Reply:
Okay, based on this conversation I'm going to merge this in order to mitigate failing jobs and when we implement a better overall design, this code will be inherently revisited.

@maestromac What about implementing a listener on the logs so we get notified of any spikes in this log output?

Reply:
yep I can do that